src/components/publishing/header/feature_header.tsx

```diff
@@ -215,15 +215,15 @@ const FeatureHeaderContainer = Div.extend`
   height: 100vh;
   &[data-type='text'] {
     height: auto;
-    .feature__title {
+    ${Title} {
```

Comment:
Were these referring to the same thing but previously in a roundabout way?

Reply:
Yeah, they are the same div

Reply:
Ah ok, so where was that classname coming from? Is that what styled-components generated?

Reply:
Originally they came from the [header component](https://github.com/artsy/reaction/blob/master/src/components/publishing/header/header.tsx), the title class has already been removed.  The deck will probably be removed too in a future PR.

Reply:
Gotcha, thanks 👍